### PR TITLE
Fix message error in nin view

### PR DESCRIPTION
**Pull request by cristeahub**
The menu displayed wrong information when it comes to the key bindings.

### CODE REVIEW

**Comment by stianjensen** [comment]
I like having forward/backward as two keys next to eachother, though.

**Comment by cristeahub** [comment]
Ye, I'll work on having editable key bindings in another prq.

**Comment by sigvef** [comment]
Closing this in anticipation of the other prq.

**Comment by cristeahub** [comment]
Oh, the other prq was thought of as a separate thing, since this prq acutally fixes an error displayed in nin.

**Comment by stianjensen** [comment]
The first commit fixes an error in nin

**Comment by cristeahub** [comment]
> The first commit fixes an error in nin

Yes

**Comment by stianjensen** [comment]
I can merge this if it contains only the first commit ^^

**Comment by cristeahub** [comment]
> I can merge this if it contains only the first commit ^^

Oh, you would like it to be two different prqs?

**Comment by stianjensen** [comment]
If we're getting user customizable keybindings, I don't see any reason to make this particular keybind change now.

**Comment by cristeahub** [comment]
> If we're getting user customizable keybindings, I don't see any reason to make this particular keybind change now.

Sure.

**Comment by cristeahub** [comment]
ping

File: nin/frontend/app/scripts/controllers/main.js [hunk 1]
```diff
@@ -23,10 +23,10 @@
             {name: 'Forward one second', shortcut: ',', click: function() {
               commands.jog(60);
             }},
-            {name: 'Rewind 10 seconds', shortcut: 'L', click: function() {
+            {name: 'Rewind 10 seconds', shortcut: 'K', click: function() {
               commands.jog(-60 * 10);
             }},
-            {name: 'Foward 10 seconds', shortcut: 'K', click: function() {
+            {name: 'Foward 10 seconds', shortcut: 'L', click: function() {
```
Comment (stianjensen) [inline nin/frontend/app/scripts/controllers/main.js]: While you're at it, there's also a typo here: 'Foward'

**Comment by cristeahub** [comment]
ping

**Pull request merged**